### PR TITLE
Added titleColorForMenu and ARC compatibility

### PR DESCRIPTION
**Pull request by bobgodwinx**
Added titleColorForMenu and ARC compatibility

### CODE REVIEW

**Comment by bobgodwinx** [comment]
forGround color for UIButton

File: MKHorizMenu/MKHorizMenu.h [hunk 1]
```diff
@@ -36,23 +37,16 @@
...
-    __weak id <MKHorizMenuDelegate> itemSelectedDelegate;
-}
-
-@property (nonatomic, retain) NSMutableArray *titles;
+@property (nonatomic) NSArray *titles;
 @property (nonatomic, weak) IBOutlet id <MKHorizMenuDelegate> itemSelectedDelegate;
-@property (nonatomic, retain) IBOutlet id <MKHorizMenuDataSource> dataSource;
-@property (nonatomic, retain) UIImage *selectedImage;
+@property (nonatomic, weak) IBOutlet id <MKHorizMenuDataSource> dataSource;
+@property (nonatomic, weak) UIImage *selectedImage;
```
Comment (neoshuo) [inline MKHorizMenu/MKHorizMenu.h]: retain should be "strong" in ARC. assign = weak.

File: MKHorizMenu/MKHorizMenu.h [hunk 2]
```diff
@@ -36,23 +37,16 @@
...
 
-    int _itemCount;
-    UIImage *_selectedImage;
-    NSMutableArray *_titles;
-    id <MKHorizMenuDataSource> dataSource;
-    __weak id <MKHorizMenuDelegate> itemSelectedDelegate;
-}
-
-@property (nonatomic, retain) NSMutableArray *titles;
+@property (nonatomic) NSArray *titles;
```
Comment (neoshuo) [inline MKHorizMenu/MKHorizMenu.h]: array needs "copy" as storage modifier

File: MKHorizMenuDemo/MKHorizMenuDemoAppDelegate.m [hunk 3]
```diff
@@ -63,11 +63,11 @@ - (void)applicationWillTerminate:(UIApplication *)application
      */
 }
 
-- (void)dealloc
-{
-    [_window release];
-    [_navigationController release];
-    [super dealloc];
-}
+//- (void)dealloc
```
Comment (neoshuo) [inline MKHorizMenuDemo/MKHorizMenuDemoAppDelegate.m:66]: plz delete comment (unnecessary)

File: MKHorizMenu/MKHorizMenu.h [hunk 4]
```diff
@@ -36,23 +39,15 @@
...
-}
+@interface MKHorizMenu : UIScrollView
 
-@property (nonatomic, retain) NSMutableArray *titles;
 @property (nonatomic, weak) IBOutlet id <MKHorizMenuDelegate> itemSelectedDelegate;
-@property (nonatomic, retain) IBOutlet id <MKHorizMenuDataSource> dataSource;
-@property (nonatomic, retain) UIImage *selectedImage;
-@property (nonatomic, assign) int itemCount;
+@property (nonatomic, weak) IBOutlet id <MKHorizMenuDataSource> dataSource;
+@property (nonatomic, weak) UIImage *selectedImage;
```
Comment (neoshuo) [inline MKHorizMenu/MKHorizMenu.h]: Please use strong.

File: MKHorizMenuDemo.xcodeproj/xcuserdata/bob.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (neoshuo) [inline MKHorizMenuDemo.xcodeproj/xcuserdata/bob.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist]: please git delete & ignore everything below  `MKHorizMenuDemo.xcodeproj/xcuserdata`

**Pull request merged**